examples/HotelAds/AddHotelAd.cs

```diff
@@ -289,7 +289,7 @@ private static string AddCampaignBudget(GoogleAdsClient client, long customerId)
                 },
                 // Set the ad group.
                 AdGroup = adGroupResourceName,
-                Status = AdGroupAdStatus.Paused
+                Status = AdGroupAdStatus.Enabled
```

Comment:
Could you please leave a comment why this has to be "Enabled"? Our code examples typically leave this as "Paused", so it is important to give the users a context on why that won't work here.

Reply:
Fixed.